client/src/components/app.jsx

```diff
@@ -10,7 +10,7 @@ function App() {
 
   return (
     <div>
-      <Overview product={product} styles={styles} setStyles={setStyles} meta={meta} />
+      <Overview product={product} styles={styles} setStyles={setStyles} metaData={meta} />
```

Comment:
I felt that metaData was more representative of the actual data. Thoughts?

Reply:
Good point, I agree